comfyui-truss/README.md

```diff
@@ -0,0 +1,312 @@
+## ComfyUI Truss
+
+This truss is designed to allow ComfyUI users to easily convert their workflows into a production grade API service. 
+
+## Exporting the ComfyUI workflow
+
+Inside ComfyUI, you can save workflows as a JSON file. However, the regular JSON format the ComfyUI uses will not work. Instead, the workflow has to be saved in the API format. Here is how you can do that:
+
+1. Go to ComfyUI and click on the gear icon for the project
+![gear_icon](images/comfyui-screenshot-1.png)
```

Comment:
Use H3 instead of a list here because the list styles are making the image alignment weird

Reply:
Hi @philipkiely-baseten, thanks for your suggestions! I've modified the Readme based on your feedback. I would love to collaborate with you on a blog post for this once Bola gives the green light.